docs/conventions.asciidoc

```diff
@@ -0,0 +1,76 @@
+//[[ecs-conventions]]
```

Comment:
Is this commented out on purpose? Probably in the original already like this.

Reply:
It's like that in the original. We'll look at this on the final push for the docs, after this release.